build/extension_content_script.js

```diff
@@ -14,8 +15,62 @@ function injectScript( filePath, tag ) {
 	node.appendChild( script );
 }
 
+/**
+ * Consider whether to activate the initial welcome tour
+ * and listen to dismissal event if so. If dismissal event
+ * happened, store that state so the welcome tour is never
+ * shown again.
+ */
+function activateWelcomeTour() {
+	theBrowser.storage.sync.get( [ 'WelcomeTourSeen' ], function ( result ) {
```

Comment:
I think there might be some documentation confusion about this, because [according to MDN](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/storage/StorageArea/get) this should be more like `.get( [ 'WelcomeTourSeen' ] ).then( function ( result ) { ... } );` — but the callback as the 2nd arg seems to work too!

I'd still say maybe go with the Promise structure, because then it's also possible to add an error handler in (which is what I did to find the addon ID issue I was having).